CMakeLists.txt

```diff
@@ -119,6 +119,11 @@ endif() # TRITON_ENABLE_GPU
 configure_file(src/libtriton_tensorflow.ldscript libtriton_tensorflow.ldscript COPYONLY)
 
 if (${TRITON_TENSORFLOW_DOCKER_BUILD})
+  if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
```

Comment:
Can we directly use ${CMAKE_HOST_SYSTEM_PROCESSOR}?

Reply:
Yes. I wanted to keep it consistent with [pytorch_backend](https://github.com/triton-inference-server/pytorch_backend/blob/main/CMakeLists.txt#L129)

I liked how it was done there since it is was more readable in my opinion. It also makes it clear to a user that only aarch64 and x86_64 are supported.